### PR TITLE
docs: improve install script docs

### PR DESCRIPTION
**Pull request by ruben-arts**
Improve the documentation on the installation script. Idea for improving sparked by @pavelzw in https://github.com/prefix-dev/pixi/pull/1129#issuecomment-2041184670

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -111,28 +107,56 @@ cargo test
...
+
+    The installation script has several options that can be manipulated through environment variables.
+
+    | Variable             | Description                                                                        | Default Value         |
+    |----------------------|------------------------------------------------------------------------------------|-----------------------|
+    | `VERSION`            | The version of pixi getting installed, can be used to up- or down-grade.           | `latest`              |
+    | `PIXI_HOME`          | The location of the binary folder.                                                  | `$HOME/.pixi`         |
+    | `PIXI_ARCH`          | The architecture the pixi version was built for.                                    | `uname -m`            |
+    | `PIXI_NO_PATH_UPDATE`| If set the `$PATH` will not be updated to add `pixi` to it.                         |                       |
+    | `TMP_DIR`            | The temporary directory the script uses to download to and unpack the binary from.   | `/tmp`                |
```
Comment (pavelzw) [inline docs/index.md:122]: not that it matters 🤔 

image
```suggestion
    | `VERSION`            | The version of pixi getting installed, can be used to up- or down-grade.           | `latest`              |
    | `PIXI_HOME`          | The location of the binary folder.                                                 | `$HOME/.pixi`         |
    | `PIXI_ARCH`          | The architecture the pixi version was built for.                                   | `uname -m`            |
    | `PIXI_NO_PATH_UPDATE`| If set the `$PATH` will not be updated to add `pixi` to it.                        |                       |
    | `TMP_DIR`            | The temporary directory the script uses to download to and unpack the binary from. | `/tmp`                |
```

File: docs/index.md [hunk 2]
```diff
@@ -111,28 +107,56 @@ cargo test
...
+pixi self-update
+```
+Or get a specific pixi version using:
+```shell
+pixi self-update --version x.y.z
+```
+
 !!! note
     If you used a package manager like `brew`, `mamba`, `conda`, `paru` to install `pixi`.
     Then use their builtin update mechanism. e.g. `brew upgrade pixi`.
```
Comment (tdejager) [inline docs/index.md:162]:
```suggestion
    It's preferable to use the built-in update mechanism. e.g. `brew upgrade pixi`.
```

File: docs/index.md [hunk 3]
```diff
@@ -111,31 +107,59 @@ cargo test
...
-    ```
-    Or get a specific pixi version using:
-    PowerShell:
+
+    The installation script has several options that can be manipulated through environment variables.
+
+    | Variable         | Environment variable | Description                                                                       | Default Value               |
+    |------------------|----------------------|-----------------------------------------------------------------------------------|-----------------------------|
+    | `PixiVersion`    | `PIXI_VERSION`       |The version of pixi getting installed, can be used to up- or down-grade.         | `latest`                    |
+    | `PixiHome`       | `PIXI_HOME`          | The location of the installation.                                                 | `$Env:USERPROFILE\.pixi`   |
```
Comment (pavelzw) [inline docs/index.md:140]:
```suggestion
    | `PixiVersion`    | `PIXI_VERSION`       |The version of pixi getting installed, can be used to up- or down-grade.         | `latest`                      |
    | `PixiHome`       | `PIXI_HOME`          | The location of the installation.                                                 | `$Env:USERPROFILE\.pixi`    |
```

File: docs/index.md [hunk 4]
```diff
@@ -111,31 +107,59 @@ cargo test
...
-    iwr -useb https://pixi.sh/install.ps1 | iex
-    ```
-    Or get a specific pixi version using:
-    PowerShell:
+
+    The installation script has several options that can be manipulated through environment variables.
+
+    | Variable         | Environment variable | Description                                                                       | Default Value               |
+    |------------------|----------------------|-----------------------------------------------------------------------------------|-----------------------------|
+    | `PixiVersion`    | `PIXI_VERSION`       |The version of pixi getting installed, can be used to up- or down-grade.         | `latest`                      |
```
Comment (ruben-arts) [inline docs/index.md:139]:
```suggestion
    | `PixiVersion`    | `PIXI_VERSION`       |The version of pixi getting installed, can be used to up- or down-grade.           | `latest`                      |
```

File: docs/index.md [hunk 5]
```diff
@@ -111,31 +107,59 @@ cargo test
...
-    iwr -useb https://pixi.sh/install.ps1 | iex
-    ```
-    Or get a specific pixi version using:
-    PowerShell:
+
+    The installation script has several options that can be manipulated through environment variables.
+
+    | Variable         | Environment variable | Description                                                                       | Default Value               |
+    |------------------|----------------------|-----------------------------------------------------------------------------------|-----------------------------|
+    | `PixiVersion`    | `PIXI_VERSION`       |The version of pixi getting installed, can be used to up- or down-grade.           | `latest`                      |
```
Comment (ruben-arts) [inline docs/index.md:139]:
```suggestion
    | `PixiVersion`    | `PIXI_VERSION`       |The version of pixi getting installed, can be used to up- or down-grade.           | `latest`                    |
```

**Pull request merged**